processor/pom.xml

```diff
@@ -92,6 +92,17 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <MapStruct-Generated-Comment>${project.groupId}:${project.artifactId}:${project.version}</MapStruct-Generated-Comment>
```

Comment:
I don't think we need group/artifact id; when seeing the version number in `@Generated` that info can be derive. I'd just use the standard version number manifest header which should already be in there.